portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,67 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+  text-align: center;
+}
+
+body {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-align: justify;
+}
+
+.content-container {
+  max-width: 650px;
+}
+
+div {
+  width: 800px;
+  border: 25px solid;
+  border-color: rgba(50, 100, 230, 0.0);
+  padding: 50px;
+  margin: 0 50px;
+}
+
+
+table {
+  margin-left:auto;
+  margin-right:auto;
+}
+
+th {
+  text-align: left;
+}
+
+th, td {
+  padding: 5px 12px;
+}
+
+tr.gallery-row {
+  height: 300px;
+  max-height: 300px;
+}
+
+figure {
+  display: block;
+  padding: 0;
+  border: 0;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+figure figcaption {
+  text-align: center;
+}
+
+.gallery-image {
+  max-height: 240px;
+  height:auto;
+  width:auto;
+  padding: 0px;
```

Comment:
Avoid specifying units for 0 values unless required.

https://google.github.io/styleguide/htmlcssguide.html#0_and_Units

Reply:
Done.